src/test/javascript/spec/component/lecture/lecture.component.spec.ts

```diff
@@ -228,4 +241,23 @@ describe('Lecture', () => {
         expect(lectureComponent.filteredLectures).toBeArrayOfSize(1);
         expect(lectureComponent.filteredLectures).toContainEqual(unspecifiedLecture);
     });
+
+    it('should sort rows', () => {
```

Comment:
Technically, you could to one test only but parametrised.
1. Parameter: predicate
2. Parameter: direction
3. Parameter: array of expected sorting
This way you can easily go through all options and make sure everything works as intended